policy/policy_file.go

```diff
@@ -503,42 +503,46 @@ func (self *Policy) Is_Version(v string) bool {
 
 func (self *Policy) String() string {
 	res := ""
-	res += fmt.Sprintf("Name: %v Version: %v, Pattern: %v\n", self.Header.Name, self.Header.Version, self.PatternId)
-	res += "API Specifications\n"
+	res += fmt.Sprintf("Name: %v, Version: %v, Pattern: %v, ", self.Header.Name, self.Header.Version, self.PatternId)
```

Comment:
getting rid of the new lines really makes this long log entry very hard to read

Reply:
It was always a long log entry because it was not calling the .String() method to begin with. Now with this change, it does call .String(), but if the newlines are in there, it won't print anything after the first newline occurrence.